contrib/platform/src/com/sun/jna/platform/win32/Winspool.java

```diff
@@ -160,6 +160,11 @@ public interface Winspool extends StdCallLibrary {
     public static final int PRINTER_NOTIFY_TYPE = 0x00;
     public static final int JOB_NOTIFY_TYPE = 0x01;
 
+    public static final short PRINTER_CONTROL_PAUSE = 0x01;
+    public static final short PRINTER_CONTROL_PURGE = 0x02;
+    public static final short PRINTER_CONTROL_RESUME = 0x03;
+    public static final short PRINTER_CONTROL_SET_STATUS = 0x04;
```

Comment:
This is only ever used in `int` context. I don't see a reason to declare it different.